### PR TITLE
[server][bugfix] Fix OGC test getfeatureinfo:each-queryable-layer

### PR DESCRIPTION
**Pull request by pblottiere**
## Description

This PR fixes the OGC test on queryable layer which is currently failing: http://37.187.164.233/ogc/10_05_2017_00_55_08_wms_1_3_0.html#c7c9de2b-94fd-49c6-8e70-a13d67447cd9/d1e17334_1/d1e17422_1/d1e9517_1/d1e9608_1

I updated unit tests accordingly.

## Checklist

- [x] Commit messages are descriptive and explain the rationale for changes
- [ ] Commits which fix bugs include `fixes #11111` in the commit message next to the description
- [ ] Commits which add new features are tagged with `[FEATURE]` in the commit message
- [ ] Commits which change the UI or existing user workflows are tagged with `[needs-docs]` in the commit message and containt sufficient information in the commit message to be documented
- [x] I have read the [QGIS Coding Standards](https://docs.qgis.org/testing/en/docs/developers_guide/codingstandards.html) and this PR complies with them
- [x] This PR passes all existing unit tests (test results will be reported by travis-ci after opening this PR)
- [x] New unit tests have been added for core changes
- [x] I have run [the `scripts/prepare-commit.sh` script](https://github.com/qgis/QGIS/blob/master/.github/CONTRIBUTE.md#contributing-to-qgis) before each commit

### CODE REVIEW

File: tests/src/python/test_qgsserver.py
Comment (rldhont) [inline tests/src/python/test_qgsserver.py]: I have also proposed some enhancement here https://github.com/qgis/QGIS/pull/5297/files#diff-260b51384025856fac2dc9682481a865

**Comment by pblottiere** [comment]
@rldhont Do you want me to change something in this PR or do you agree with its content?

**Comment by rldhont** [comment]
@pblottiere I am agree with your PR, but can you take a look at the changes I have done https://github.com/qgis/QGIS/pull/5297/files#diff-260b51384025856fac2dc9682481a865 you can probably help me to enhance it.

**Comment by pblottiere** [comment]
I'll merge this PR in the coming days if no more comments come up.

**Comment by rldhont** [comment]
Before merging I'd like to homogenize the `test_qgsserver.py` code in this and #5297 PR.

**Comment by pblottiere** [comment]
@rldhont 

> Before merging I'd like to homogenize the test_qgsserver.py code in this and #5297 PR

I homogenised my tests with your modifications as you requested. I'll merge this PR tomorrow if you have no more comments in the meantime.

**Comment by rldhont** [comment]
@pblottiere can you rebase!

**Review by rldhont — APPROVED** [review]
You can merge

**Comment by pblottiere** [comment]
Corresponding OGC test is fixed now :  http://test.qgis.org/2017_10_17_14_28_wms_1_3_0.html#da44997d-1ab7-4dfa-b2c6-41a372eb9361/d1e17334_1/d1e17422_1/d1e9517_1

**Pull request merged**
